packages/shared/src/db/schema.ts

```diff
@@ -725,6 +725,17 @@ export const posts = sqliteTable(
   })
 );
 
+export const hiddenPosts = sqliteTable(
```

Comment:
Would it simplify things to just have an `is_hidden` field on the posts table? Or do we need this.

Reply:
I think we can just use the is_hidden field and make sure it doesn't get overwritten